### PR TITLE
Update Assigning-Labels doc

### PR DESCRIPTION
**Pull request by jheysel-r7**
Not the most impactful documentation update ever made however I noticed this file was pretty out of date. I've updated to reflect some of the more used labels and attempted to alphabetize the list. I left out labels that I thought were no longer applicable to PRs.

### CODE REVIEW

**Comment by adfoster-r7** [comment]
## Release Notes

Documents the latest labels that can be assigned to pull requests

File: docs/metasploit-framework.wiki/Assigning-Labels.md [hunk 1]
```diff
@@ -49,4 +73,74 @@ Because of how GitHub tracks changes between branches and what got added in a pa
 git checkout -b <BRANCH_NAME>
 git push <your_fork_remote> <BRANCH_NAME>
 ```
-This helps protect the process, ensure users are aware of commits on the branch being considered for merge, allows for a location for more commits to be offered without mingling with other contributor changes and allows contributors to make progress while a PR is still being reviewed.
\ No newline at end of file
+This helps protect the process, ensure users are aware of commits on the branch being considered for merge, allows for a location for more commits to be offered without mingling with other contributor changes and allows contributors to make progress while a PR is still being reviewed.
+
+### Needs-docs
+
+When a module is uploaded without a corresponding documentation file, add this label in indicate docs are required
```
Comment (tekwizz123-mobile) [inline docs/metasploit-framework.wiki/Assigning-Labels.md:80]: s/in/to

File: docs/metasploit-framework.wiki/Assigning-Labels.md [hunk 2]
```diff
@@ -26,19 +50,19 @@ Plugins and scripts, anything that's not otherwise defined.
 
 ### Module
 
-Touches something in /modules
+Touches something in /modules.
 
-### Specs
```
Comment (tekwizz123-mobile) [inline docs/metasploit-framework.wiki/Assigning-Labels.md:31]: Out of curiosity do we still use this tag at all? I've been thinking of writing more specs recently so not sure if this tag would be beneficial or if it's related to old behavior?

Comment (adfoster-r7) [reply docs/metasploit-framework.wiki/Assigning-Labels.md:31]: `specs` isn't a label that exists currently; there's a `tests` label that's occasionally used, but it's not wired into any automated processes though so it's not particularly noteworthy (in my opinion)

Comment (tekwizz123-mobile) [reply docs/metasploit-framework.wiki/Assigning-Labels.md:31]: Makes sense thanks for clarifying 👍

**Pull request merged**